pcweb/pcweb.py

```diff
@@ -23,7 +23,7 @@
         route.path,
         "Reflex",
         description="Performant, customizable web apps in pure Python. Deploy in seconds.",
-        image="preview.png",
+        image="/index_preview.png",
```

Comment:
```suggestion
        image="previews/index_preview.png",
```
This one is nlot like the others

Reply:
Fixed in my other pushes it think